### PR TITLE
feat: migrate tag recipes to the new structure & format

### PR DESCRIPTION
**Pull request by toph-allen**

### CODE REVIEW

File: 06-content-organization/03-creating-tags/index.qmd [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+## Discussion
+
+Tags on Connect can be nested arbitrarily deep. When creating a tag, you can specify a parent to add the new tag underneath it in the hierarchy.
+
+::: {.panel-tabset group="language"}
+
+## Python
+
+Specify the parent tag with its will need the identifier. See [Viewing Tag Information](../01-viewing-tag-information/index.qmd) for details.
```
Comment (tdstein) [inline 06-content-organization/03-creating-tags/index.qmd:78]: Some grammar issues here. Not sure what it should say.

**Review by tdstein — APPROVED** [review]
found one typo question, which might be carry over from the previous implementation. otherwise lgtm

File: 06-content-organization/03-creating-tags/index.qmd (hunk 1 above)
Comment (toph-allen) [reply 06-content-organization/03-creating-tags/index.qmd:78]: I'll fix it!

Comment (toph-allen) [reply 06-content-organization/03-creating-tags/index.qmd:78]:
```suggestion
Specify the parent tag with its `id`. See [Viewing Tag Information](../01-viewing-tag-information/index.qmd) for details.
```

**Pull request merged**